File: Spring/Chapter7/MyBatisSpringJavaInit/src/test/java/manyToMany/ManyToManyTest.java
package manyToMany;

import org.junit.jupiter.api.Test;
import org.springframework.context.ApplicationContext;
import org.springframework.context.support.ClassPathXmlApplicationContext;
import pers.manyToMany.dao.OrdersWithProductDao;

public class ManyToManyTest {
    @Test
    public void test()
    {
        ApplicationContext context = new ClassPathXmlApplicationContext("applicationContext.xml");
        OrdersWithProductDao dao = context.getBean(OrdersWithProductDao.class);
        System.out.println(dao.selectOrdersAndProduct());
    }
}

File: Spring/Chapter7/MyBatisSpringKotlinInit/src/test/kotlin/manyToMany/ManyToManyTest.kt
package manyToMany

import org.junit.Test
import org.springframework.context.support.ClassPathXmlApplicationContext
import pers.manyToMany.dao.OrdersWithProductDao

class ManyToManyTest {
    @Test
    fun test()
    {
        val context = ClassPathXmlApplicationContext("applicationContext.xml")
        val dao = context.getBean(OrdersWithProductDao::class.java)
        println(dao.selectOrdersAndProduct())
    }
}